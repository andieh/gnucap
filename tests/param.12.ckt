spice
.options numdgt 10
.param a=3.141592653
.eval a
.options numdgt 5
.eval a
