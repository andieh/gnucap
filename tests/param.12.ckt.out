spice 
a= 3.141592653
a= 3.1416
